### PR TITLE
acronym: update tests to v1.6.0

### PR DESCRIPTION
**Pull request by GascaK**
Updated test case for apostrophes in [canonical data](https://github.com/exercism/problem-specifications/blob/master/exercises/acronym/canonical-data.json) and updated example.py to correctly solve the new apostrophe test case.

### CODE REVIEW

File: exercises/acronym/acronym_test.py [hunk 1]
```diff
@@ -31,6 +31,9 @@ def test_consecutive_delimiters(self):
         self.assertEqual(
             abbreviate('Something - I made up from thin air'), 'SIMUFTA')
 
+    def test_apostrophes(self):
+        self.assertEqual(abbreviate('Halley\'s Comet'), 'HC')
```
Comment (cmccandless) [inline exercises/acronym/acronym_test.py]: For readability, can you please use double quotes around the input string to negate the need for escaping the apostrophe?

File: exercises/acronym/example.py [hunk 2]
```diff
@@ -2,5 +2,5 @@
 
 
 def abbreviate(words):
-    regex = '[A-Z]+[a-z]*|[a-z]+'
+    regex = '[A-Z]+[\'a-z]*|[\'a-z]+'
```
Comment (cmccandless) [inline exercises/acronym/example.py]: Again, switching to double quotes for this string literal removes the need for escaping apostrophes.

**Comment by GascaK** [comment]
Yes, makes sense I will get on that later tonight when I get some free time.

**Comment by cmccandless** [comment]
Merged; thanks for working on this!

**Pull request merged**